### PR TITLE
HoC 2023 - Update How-to guides • Districts

### PR DESCRIPTION
**Pull request by kelbyhawn**
Update Districts How-to guide: https://hourofcode.com/how-to/districts
- Page is a draft: can be seen locally at http://localhost.hourofcode.com:3000/us/en/how-to/districts-draft
- Will update each How-to page as a draft and set everything live in a separate PR

**Related PRs:**
- https://github.com/code-dot-org/code-dot-org/pull/54685
- https://github.com/code-dot-org/code-dot-org/pull/54720

**Jira ticket:** [ACQ-1009](https://codedotorg.atlassian.net/browse/ACQ-1009)

----

localhost hourofcode com_3000_us_how-to_districts-draft

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/views/how-to/faq/faq_default.haml
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/views/how-to/faq/faq_default.haml:1]: These two questions will be reused on multiple pages so I made this view more evergreen.

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -8603,6 +8617,13 @@
...
+  hoc_how_to_step_districts_promote_desc_01: "Let community members and stakeholders know about your participation in Hour of Code Events! Post to Facebook, Twitter, Instagram and your district's homepage! Explore our sample emails which help you share your Hour of Code event with your community."
+  hoc_how_to_step_districts_check_out_ai_desc_01: "Demystify artificial intelligence (AI) by learning how it's changing the ways we learn and engage our students. \"How AI Works\" provides opportunities for districts to deepen their understanding of AI through professional learning opportunities such as, \"A1 101 for Teachers\" and through [lessons on How AI Works](%{lessons_url}). These tools will help districts, teachers and students learn how to more effectively understand and utilize AI."
```
Comment (TurnerRiley) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: I think this says "A1 101" instead of "AI 101"?

**Review by TurnerRiley — APPROVED** [review]
Looks great! Just 1 quick typo fix

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Fixed!

**Pull request merged**